tslint.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
FYI tslint is deprecated and eslint with typescript plugin is what should be used for new projects